tox.ini

```diff
@@ -1,14 +1,16 @@
 [tox]
+minversion = 2.4.0
 envlist =
     py27,py34,py35,py36,py37,pypy,pypy3,flake8
 
 [testenv]
-passenv = LANG LC_CTYPE LC_ALL MSYSTEM
+passenv = SKIP_NO_TESTS
```

Comment:
Hey, why haven't I noticed this before?  This could also be causing Windows test breakages?

Reply:
Do you know if the pytest issue is documented in an issue that could follow?

Reply:
The pytest bug is here: https://github.com/pytest-dev/pytest/issues/4389

Reply:
Thanks!